src/geocat/viz/taylor/taylor.py

```diff
@@ -0,0 +1,600 @@
+import numpy as np
+import matplotlib.pyplot as plt
+
+
+class TaylorDiagram(object):
+    """Taylor Diagram.
+
+    Taylor diagrams provide a visual framework for comparing a suite of variables from one or more test data sets to
+    one or more reference data sets. Commonly, the test data sets are model experiments while the reference data set
+    is a control experiment or some reference observations (eg, ECMWF Reanalyses). Generally, the plotted values are
+    derived from climatological monthly, seasonal or annual means. Because the different variables (eg:
+    precipitation, temperature) may have widely varying numerical values, the results are normalized by the reference
+    variables. The ratio of the normalized variances indicates the relative amplitude of the model and observed
+    variations.
+
+    References
+    ----------
+    - https://validate-climate-model-validation.readthedocs.io/en/latest/_modules/validate/taylor.html
+    - https://matplotlib.org/stable/gallery/axisartist/demo_floating_axes.html#sphx-glr-gallery-axisartist-demo-floating-axes-py
+    - https://www.ncl.ucar.edu/Applications/taylor.shtml
+    """
+
+    def __init__(self,
+                 refstd=1,
+                 fig=None,
+                 rect=111,
+                 label='REF',
+                 stdRange=(0, 1.65),
+                 stdLevel=np.arange(0, 1.51, 0.25)):
+        """Create base Taylor Diagram.
+
+        Parameters
+        ----------
+        refstd : float, optional
+            reference standard deviation
+
+        fig : matplotlib.figure.Figure, optional
+            Optional input figure. Default is None
+
+        rect : int, optional
+            Optional subplot definition
+
+        label : string, optional
+            Optional reference label string indentifier
+
+        stdRange : Tuple, optional
+            Optional stddev axis extent
+
+        stdLevel : list, optional
+            Optional list of tick locations for stddev axis
+        """
+
+        from matplotlib.projections import PolarAxes
```

Comment:
I think these imports should all be in one place at the top with the other imports